### ISSUE TITLE
[Feature request] Automatically hide bottom progress panel as well as mouse pointer 

### ISSUE THREAD

**Issue by RomanSedykh** [johnfactotum/foliate#230]
With 16:9 screens vertical space is more valuable, so additional space for text will be way more important than constantly seeing that panel.

![Screenshot_2020-01-12_14-32-50](https://user-images.githubusercontent.com/59629258/72218123-7714f100-3548-11ea-9740-feed1ced3a26.png)

**Comment by johnfactotum** [comment]
On 1.5.3 you can use Ctrl+P to toggle the bar.

On master the bottom bar autohides and a footer is shown instead. See https://github.com/johnfactotum/foliate/issues/175#issuecomment-553254359. However currently even if you set both footer texts to nothing, it will still display an empty bar, and the plan is to make it an overlaid bar like the headerbar when the footer is empty.

**Comment by digitalethics** [comment]
> the plan is to make it an overlaid bar like the headerbar when the footer is empty.

Currently, when in skeuomorphic mode the bottom bar (unfortunately) docks onto the skeuomorphic margins. Do I understand it correctly that you plan to move it further below so that it creates a functional and aesthetic balance to the header bar?

**Comment by johnfactotum** [comment]
No, overlaid as in how the headerbar works in fullscreen. When both left and right footer are set to "Nothing", there's no point in displaying an empty bar. It should not show any bar at all, and the navbar will be _overlaid_ on top of the page.

> aesthetic balance to the header bar

I think to achieve balance one should move the headerbar inside the skeuomorphic area.
